scripts/in_container/run_resource_check.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import psutil
+from rich.console import Console
+
+console = Console(force_terminal=True, color_system="standard", width=180)
+
+
+def get_size(kilobytes):
+    """
+    Convert kilobytes into gigabytes
+    1 Gigabytes = 1,048,576 Kb
+    """
+    factor = 1000000000
+    value_gb = kilobytes // factor
+    return value_gb
+
+
+def resoure_check():
+    """
+    Use gsutil to get resources in kylobytes: memory, cpus and disk
+    """
+    resources = {}
+    print("\nChecking resources.\n")
+
+    # Memory available
+    svmem = psutil.virtual_memory()
+    mem_available = get_size(svmem.total)
+    resources.setdefault('Memory', []).append(mem_available)
+
+    # Cpus available
+    cpus_available = psutil.cpu_count(logical=True)
+    resources.setdefault('Cpus', []).append(cpus_available)
+
+    # Disk: Get all disk partitions /
+    partitions = psutil.disk_partitions()
+    partition_usage = psutil.disk_usage(partitions[0].mountpoint)
+    disk_available = get_size(partition_usage.free)
+    resources.setdefault('Disk', []).append(disk_available)
+
+    return resources
+
+
+def resoure_validate():
+    resources = resoure_check()
+    warning_resources = False
+    check = "OK"
+
+    resources.setdefault("Memory", []).append(4)
+    resources.setdefault("Cpus", []).append(2)
+    resources.setdefault("Disk", []).append(20)
+
+    for resource, available in resources.items():
+
+        check = '' if resource == "Cpus" else 'GB'
+
+        if (
+            resource == "Memory"
+            and available[0] < 4
```

Comment:
The numbers (4,2,20) are repeated here and above - they should be extracted as constants.

Reply:
Should  I make the same with "Memory", "Cpus" and "Disk", they are used in both functions, the global variable will be good here?